### ISSUE TITLE
How long should train_online take?

### ISSUE THREAD

**Issue by hanrelan** [kmaninis/OSVOS-PyTorch#14]
Using the default parameters with the "blackswan" sequence, train_online.py takes about 30 minutes for fine tuning + processing the test sequence on the GTX 1080 Ti. Is that normal?

Also as a quick follow up - to test with multiple frames annotated would I simply change the dataloader to load more than one frame when in test mode?

Thanks for sharing this!

**Comment by kmaninis** [comment]
Hi, you are right! You can control the running time by decreasing the number of iterations (and sacrifice quality).

If you want to use extra frames for fine-tuning, yes, you only need to change the images that the dataloader reads, but in training mode of train_online.
